concepts/40 Angular Components/30 Application Template/60 Internet Explorer 11 Support.md

```diff
@@ -0,0 +1,41 @@
+For the generated application to work in Internet Explorer 11, make the following adjustments:
```

Comment:
I would rewrite this a bit:
Specify the following configuration options to make the generated application work in Internet Explorer 11:

Reply:
These are not really options. How about this?

```suggestion
To make the generated application work in Internet Explorer 11, do the following:
```

Reply:
LGTM